setup.py

```diff
@@ -15,11 +18,22 @@
     "moto==1.3.13",
 ]
 
+# Import README for long-description.
+with open(here / "README.md") as f:
+    long_description = f.read()
```

Comment:
you could use: https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text